pkgs/applications/display-managers/ly/default.nix

```diff
@@ -1,20 +1,25 @@
-{ stdenv, lib, fetchFromGitHub, linux-pam }:
+{ stdenv, lib, fetchFromGitHub, linux-pam, git, libxcb, ncurses }:
+
+# This package is intended to be reworked, it is temporarily using a fork since
+# the original developer used a non-standard method of specifying .gitmodules.
+# https://github.com/nullgemm/ly/pull/279
 
 stdenv.mkDerivation rec {
   pname = "ly";
-  version = "0.2.1";
+  version = "master";
```

Comment:
you should use a more concrete version number

Reply:
This is not utilized, see the comment at the top of the file.

Reply:
I have removed it, I forgot to do so :smile: